commands/bug/bug_select.go

```diff
@@ -44,7 +44,7 @@ The complementary command is "git bug deselect" performing the opposite operatio
 
 func runBugSelect(env *execenv.Env, args []string) error {
 	if len(args) == 0 {
-		return errors.New("You must provide a bug id")
+		return errors.New("a bug id must be provided")
```

Comment:
I've been guilty of making those error messages a bit more human as they show up directly to the user. That was a long time ago though, dunno if it really makes sense.

Reply:
I tried to make it somewhat human and match the convention for error messages - your choice (the capitalization was what the linter complained about and I rearranged it to get rid of the word "You")

Reply:
Yeah, that also make sense. I suppose that to make things really proper we would need some kind of augmented error to render exactly how it should be, which would also be required  if we ever do internationalization.